tests/ui_tests/test_provider_page.py

```diff
@@ -147,3 +196,125 @@ def test_run_upgrade_on_provider_page_by_toolbar(self, app_fs, sdk_client_fs, up
+            config_rows = provider_config_page.config.get_all_config_rows()
+            assert len(config_rows) == 1, "Rows are not filtered: there should be 1 row"
+            assert (
+                provider_config_page.config.get_config_row_info(config_rows[0]).name == f"{params['search_param']}:"
+            ), f"Name should be {params['search_param']}"
+        with provider_config_page.config.wait_rows_change():
+            provider_config_page.config.clear_search_input()
+        with allure.step("Check that rows are not filtered"):
+            config_rows = provider_config_page.config.get_all_config_rows()
+            assert len(config_rows) == 4, "Rows are filtered: there should be 4 row"
+        with provider_config_page.config.wait_rows_change():
+            provider_config_page.config.click_on_group(params["group_name"])
+        with allure.step("Check that groups are closed"):
+            config_rows = provider_config_page.config.get_all_config_rows()
+            assert len(config_rows) == 2, "Groups are not closed: there should be 2 row"
+
+    @pytest.mark.smoke()
+    def test_save_custom_config_on_provider_config_page(self, app_fs, upload_and_create_test_provider):
+        params = {
+            "row_value_new": "test",
+            "row_value_old": "0000",
+            "config_name_new": "test_name",
+            "config_name_old": "init",
+        }
+        provider_config_page = ProviderConfigPage(
+            app_fs.driver, app_fs.adcm.url, upload_and_create_test_provider.id
+        ).open()
+        config_row = provider_config_page.config.get_all_config_rows()[0]
+        provider_config_page.config.type_in_config_field(
+            row=config_row, value=params["row_value_new"], clear=True, adcm_test=None
+        )
+
+        provider_config_page.config.set_description(params["config_name_new"])
+        provider_config_page.config.save_config()
+        provider_config_page.config.compare_current_to(params["config_name_old"])
+        with allure.step("Check row history"):
+            row_with_history = provider_config_page.config.get_all_config_rows()[0]
+            provider_config_page.config.wait_history_row_with_value(row_with_history, params["row_value_old"])
+
+    def test_reset_config_in_row_on_provider_config_page(self, app_fs, upload_and_create_test_provider):
+        params = {
+            "row_name": "str_param:",
+            "row_value_new": "test",
+            "row_value_old": "0000",
+            "config_name": "test_name",
+        }
+        provider_config_page = ProviderConfigPage(
+            app_fs.driver, app_fs.adcm.url, upload_and_create_test_provider.id
+        ).open()
+        config_row = provider_config_page.config.get_all_config_rows()[0]
+        provider_config_page.config.type_in_config_field(
+            row=config_row, value=params["row_value_new"], clear=True, adcm_test=None
```

Comment:
`adcm_test` is no longer an argument, this call will fall

Reply:
removed